src/backend/app/Models/AssetPerson.php

```diff
@@ -47,4 +47,8 @@ public function creator(): MorphTo {
     public function device(): BelongsTo {
         return $this->belongsTo(Device::class);
     }
+
+    public function assetType(): BelongsTo {
+        return $this->belongsTo(AssetType::class);
+    }
```

Comment:
`AssetPerson` is not linked to `AssetType` directly. This tries to solve 

```
Access to an undefined property App\Models\AssetPerson::$assetType.
```

What about:

```
$loan->assetPerson->asset->assetType->name,
```

instead?